contracts/sNOTE.sol

```diff
@@ -211,6 +211,12 @@ contract sNOTE is
         );
     }
 
+    function migrateGauge(ILiquidityGauge oldGauge) external onlyOwner {
+        _claimBAL(oldGauge);
```

Comment:
@weitianjie2000 Can we just hardcode this value in the contract?

Reply:
done